.gitignore

```diff
@@ -87,3 +87,6 @@ ENV/
 
 # Rope project settings
 .ropeproject
+
+#Pycharm Files
```

Comment:
@briancurtin may prefer not to have changes to `.gitignore` on a feature PR.

Reply:
As long as we're not solving multiple distinct problems within the actual code/tests/docs of a PR I'm ok with it. You can leave this in.